pkg/pillar/types/ifnametopci.go

```diff
@@ -89,6 +89,23 @@ func ifNameToPci(log *base.LogObject, ifName string) (string, error) {
 	return target, fmt.Errorf("Not PCI %s", target)
 }
 
+// Returns the long PCI IDs for Virtual function
+func vfIfNameToPci(ifName string) (string, error) {
```

Comment:
I think we have been overloading the `pillar/types` package, which was initially probably only meant to contain data type declarations and constants. What about moving all these SR-IOV helpers to a separate pillar/sriov package?

Reply:
That's actually good idea, I'll move out whole SRIOV functionality into pillar package.

Reply:
@milan-zededa moved it, but it might be even better to keep it in pkg/lib (since it's only set of functions, not micro service using pubSub), but I think for now it's alright to keep it as separate package